src/Codeception/Lib/Driver/Facebook.php

```diff
@@ -30,83 +38,96 @@ public function __construct($config, $logCallback = null)
      */
     public function createTestUser($name, array $permissions)
     {
+        $app_token = $this->appId . '|' . $this->appSecret;
```

Comment:
Nitpick didn't show this as an issue. And we have lots of places where we have underscores in variable names. 
There is no standard actually. But yes, `$appToken` looks better to me